### PR TITLE
JKR-7995: be compliant with Django quotes...

### PR DESCRIPTION
**Pull request by m-vdb**
merge with https://work4labs.atlassian.net/browse/JKR-7995 only

### CODE REVIEW

File: lib/storage/sfCacheSessionStorage.class.php [hunk 1]
```diff
@@ -260,6 +267,26 @@ public function shutdown()
...
+    return '"'.$value.'"';
+  }
+
+  /**
+   * Be compliant with Django, remove quotes.
+   */
+  public function deserializeCookieValue($value)
+  {
+    $matches = [];
+    if (preg_match('/"(.+)"/i', $value, $matches)) {
```
Comment (olivierlefloch) [inline lib/storage/sfCacheSessionStorage.class.php]: Woah, that's a bit overkill? Why not simply check if the first and last char of the string are double quotes and then return the adequate substring? That'll likely be significantly faster?

Comment (olasfar) [inline lib/storage/sfCacheSessionStorage.class.php]: +1

Comment (m-vdb) [inline lib/storage/sfCacheSessionStorage.class.php]: ok let's do this!

Comment (m-vdb) [inline lib/storage/sfCacheSessionStorage.class.php]: see 5caf632

**Comment by olivierlefloch** [comment]
LGTM

**Pull request merged**